modules/internal/ChapelDomain.chpl

```diff
@@ -25,6 +25,7 @@ module ChapelDomain {
   public use ChapelBase;
   use ArrayViewRankChange, ChapelTuple;
 
+  private use ChapelRange only isValidRangeIdxType;
```

Comment:
It's not unlikely that you now need to re-run a paratest and verify that this hasn't affect the module initialization order (or it has, and you need to update the module init order test)